app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<p> User name with posts </p>
\ No newline at end of file
+<h2> User name with posts </h2>
+
+<div class = 'user-content'> 
+<div class = 'user'>
+<div class = "image-placeholder"></div>
+<div class = 'user-block'>
+ <h3><%= @user.name %></h3>
+   <p > Number of posts: <%= @user.posts.count%> </p>
+ </div>
+</div>
+ <div class ='user-bio'>
+ <p><strong>Bio </strong> </p>
+ <p> <%= @user.bio %> </p>
+  </div>
+  <div class = 'posts-content'>
+     <% @posts.each do |post|%>
```

Comment:
- [ ] Hi @AmineDerbal, let's kindly filter out the 3 most recent posts(`@user.recent_posts.each`) created by each user instead of rendering all the user's posts. Remember we were asked to use the `#recent_posts` function created in the User model for this specific.

Reply:
done